lib/util/security.js

```diff
@@ -21,6 +21,9 @@ function setHiddenProperty(target, source, propertyName = 'password') {
   });
 }
 
+const hasOwnProperty = Object.prototype.hasOwnProperty;
```

Comment:
I'm not sure it's a good idea to do this reexport. the fact that it needs to be invoked via ".call" is not obvious.

helper function wpuld be clearer:

function hasOwn(obj, key) {
  return Object.prototype.hasOwnProperty.call(obj, key);
}

Reply:
I just find myself typing it so often that I wanted to make it more easy to access. This is fine too; the main thing is that I want to capture the value of `hasOwnProperty` at load time, which slightly reduces the risk that it gets monkey patched by something (or potentially exploited). I can do that and still export a function.

Once we go beyond node 16 as minimum version we can just use Object.hasOwn anyway